scss/pattern-lib.scss

```diff
@@ -1,12 +1,2 @@
-@import 'vanilla-framework/scss/vanilla';
-@include vanilla;
-
 @import 'docs-vanilla-theme/scss/theme';
-@include docs-content;
-@include docs-footer;
-@include docs-header;
-@include docs-nav;
-@include docs-notes;
-@include docs-rows;
-@include docs-wrapper;
-@include docs-toc;
+@include docs-vanilla-theme;
```

Comment:
The reason I was cherry picking the modules from docs theme is because we dont want the load the vanilla-framework that comes with docs-vanilla-theme. The site itself should include its own.